libs/platform/src/lib/components/value-help-dialog/value-help-dialog.module.ts

```diff
@@ -45,7 +45,7 @@ import { ConditionCountMessageDirective } from './directives/condition-count-mes
   imports: [
     CommonModule,
     FormsModule,
-
+BarModule,
```

Comment:
formatting issue? Also, please fix the [library imports](https://github.com/SAP/fundamental-ngx/pull/5810/files#diff-f3fb9c2e4f358ce86673535a5ce5fcb206a55bda9574a9f089814381270a8aeaR5-R25) into the split libs.

Reply:
fixed